README.md

```diff
@@ -176,6 +179,8 @@ datadog_config:
 system_probe_config:
   enabled: true
   sysprobe_socket: /opt/datadog-agent/run/sysprobe.sock
+network_config:
+  enabled: true
```

Comment:
```suggestion
system_probe_config:
  sysprobe_socket: /opt/datadog-agent/run/sysprobe.sock
network_config:
  enabled: true
```


We should avoid including system_probe_config.enabled in docs when we can, it can inadvertently enable other billed features